apps/src/code-studio/components/progress/course_progress.jsx

```diff
@@ -22,14 +24,25 @@ const styles = {
  */
 const CourseProgress = React.createClass({
   propTypes: {
+    onOverviewPage: React.PropTypes.bool.isRequired,
```

Comment:
Nit: I wonder if this name implies too much knowledge about the context in which this component is rendered. Should it be something more like `withOverviewControls`?